### PR TITLE
chore: add `Cargo.lock` + minum deposit condition inc by factor of 10

### PR DESCRIPTION
**Pull request by dj8yfo**
Cargo.lock may not be needed for learning an example, 
but contract is an application, so its dependencies generally have to be fixed by vcs

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -5,7 +5,7 @@ use near_sdk::{near, Gas, NearToken};
 mod deploy;
 mod manager;
 
-const NEAR_PER_STORAGE: NearToken = NearToken::from_yoctonear(10u128.pow(18)); // 10e18yⓃ
+const NEAR_PER_STORAGE: NearToken = NearToken::from_yoctonear(10u128.pow(19)); // 10e18yⓃ
```
Comment (dj8yfo) [inline src/lib.rs:8]: without this i wasn't seeing a message like 
```bash
Error:
   0: Error: An error occurred during a `FunctionCall` Action, parameter is debug message.
      ExecutionError("Smart contract panicked: panicked at src/deploy.rs:41:9:\nAttach at least 1.55 NEAR ")
```

**Comment by dj8yfo** [comment]
@gagdiez

File: src/lib.rs (hunk 1 above)
Comment (gagdiez) [reply src/lib.rs:8]: I don't feel the error message and this line are not related,

Comment (gagdiez) [reply src/lib.rs:8]: could you give an example with a broken test using workspaces? (i.e. a broken test on `tests/sandbox.rs` that way we could see the error and debug it easily

Comment (dj8yfo) [reply src/lib.rs:8]: @gagdiez done. added test in https://github.com/near-examples/factory-rust/pull/4/commits/d0ab63f1e9e7b7fca7d4496396403531978f9ea7 , which passes for both 18 and 19 values )))
happily, `sandbox` behaviour is consistent with `testnet` (not 146% sure about mainnet, but should be the same 1 NEAR per 100 Kb), 
but i've noticed `testnet` behaviour of this example to be confusing, when the value is 18

**Pull request closed without merging**